packages/apps/dev-wallet/src/pages/select-profile/select-profile.tsx

```diff
@@ -12,50 +20,60 @@ export function SelectProfile() {
   }
   return (
     <main>
```

Comment:
this `main` tag was removed in my PR as there is already `main` in the layout itself